test_vm/vendor/cookbooks/dbfit_test/recipes/selinux.rb

```diff
@@ -0,0 +1,9 @@
+selinux_state 'SELinux Permissive' do
+  temporary true
+  action :permissive
+end
+
+selinux_state 'SELinux Permissive' do
```

Comment:
Can we give that other name? There used it be some caveats for resources with same name (I think Chef treats is as a single entity and runs it only once).